packages/stockflux-launcher/src/toolbar/CloseButton.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import Components from 'stockflux-components';
 import { OpenfinApiHelpers } from 'stockflux-core';
+import ConfirmationWindow from '../popups/ConfirmationWindow';
 
 export default (
-  <Components.Buttons.Close
-    onClick={() => OpenfinApiHelpers.getCurrentWindowSync().close()}
-  />
+  <ConfirmationWindow
+    message="Are you sure you wish to quit?"
```

Comment:
Can this be made into a prop so that the message could be adapted in future e.g. the message for shutting down the launcher should probably be different than the message for other components, as the launcher closes everything. 

I'm not sure if we need to ask for the user to confirm close on every component also (unless some data is lost), but that's just my preference.

Reply:
This instance of the `CloseButton` is only used on the launcher close option. If you wanted to use a custom version of confirmation you would use `ConfirmationWindow` which has the property to have a custom message alongside with the `onConfirm`/`onReject` functions.

But I think the `ConfirmationWindow` and `PopupWindow` should be moved into the `stockflux-components` section, so they can be used in other sections.